pkgs/os-specific/linux/anbox/default.nix

```diff
@@ -118,8 +112,30 @@ stdenv.mkDerivation rec {
     EOF
   '';
 
+  patches = [
+    # Fixes compatibility with lxc 4
+    (fetchpatch {
+      url = "https://git.alpinelinux.org/aports/plain/community/anbox/lxc4.patch?id=64243590a16aee8d4e72061886fc1b15256492c3";
+      sha256 = "1da5xyzyjza1g2q9nbxb4p3njj2sf3q71vkpvmmdphia5qnb0gk5";
+    })
+    # Wait 10× more time when starting
+    # Not *strictly* needed, but helps a lot on slower hardware
```

Comment:
This means it is still racy though...

Reply:
I think the purpose of this patcheset is to get anbox to a more usable state. I wouldn't spend more energy on doing proper things as upstream is dead.